content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2936,6 +2936,48 @@ model User {
+
+<Admonition type="warning">
+
+**Only supported by CockroachDB** <br />
+The sequence function is for now only supported by [CockroachDB connector](/concepts/database-connectors/cockroachdb).
+
+</Admonition>
+
+Create a sequence of integers in the underlying database and assign the incremented values to the ID values of the created records based on the sequence.
+
+#### Optional arguments
+
+| Argument    | Example                                               | Description                                                                                                                                                                                                                                           |
+|-------------|-------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `virtual`   | `@default(sequence(virtual))`                         | Virtual sequences are sequences that do not generate monotonically increasing values and instead produce values like those generated by the built-in function `unique_rowid()`. They are intended for use in combination with `SERIAL`-typed columns. |
+| `cache`     | `@default(sequence(cache: 20))`                       | The number of sequence values to cache in memory for reuse in the session. A cache size of `1` means that there is no cache, and cache sizes of less than `1` are not valid.                                                                          |
+| `increment` | `@default(sequence(increment: 4, cache: 12))`         | The new value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.                                                                                                 |
+| `minValue`  | `@default(sequence(minValue: 10, maxValue: 3030303))` | The new minimum value of the sequence.                                                                                                                                                                                                                |
+| `maxValue`  | `@default(sequence(minValue: 10, maxValue: 3030303))` | The new maximum value of the sequence.                                                                                                                                                                                                                |
+| `start`     | `@default(sequence(start: 2))`                        | The value the sequence starts at if you `RESTART` or if the sequence hits the `maxValue` and `noCycle` is not set.                                                                                                                                    |
```

Comment:
`RESTART` probably has no meaning in Prisma context, but SQL only. Do we still want to mention it?

Reply:
Maybe we should rewrite these to fit better to our context...